### PR TITLE
:memo: Updated README

### PR DESCRIPTION
**Pull request by garbit**
Updated readme with installation instructions and minimum requirements.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,21 @@ A Python library for interfacing with the "ONZO" energy monitor
 
 Tested on Linux & MacOS
 
+# Installation
+This software requires that you use [Python 3.x](https://www.python.org/downloads/).
+
+To use this package you must also ensure you install hidapi.
+```
+sudo pip install hidapi
```
Comment (ep1cman) [inline README.md]: This should actually be:
`sudo pip install -r requirements.txt `

File: README.md [hunk 2]
```diff
@@ -3,6 +3,21 @@ A Python library for interfacing with the "ONZO" energy monitor
...
+```
+
+# Energy readings
+To get a basic understanding of the meter readings from your Onzo energy meter you can use ``device.py`` script to view data being broadcast over 433Mhz from your energy clamp and then read the data over usb connected to your computer.
```
Comment (ep1cman) [inline README.md]: `read.py` does the readings, `device.py` just contains the class for talking with the device.

**Review by ep1cman — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (garbit) [reply README.md]: Yeah, my bad I don't know why I used that filename. Making the changes now.

**Comment by garbit** [comment]
I've updated the master branch with the proposed changes. Please see [https://github.com/garbit/PyOnzo](https://github.com/garbit/PyOnzo) for suggested changes.

**Comment by ep1cman** [comment]
All looks good, one final request, could you squash this into a single commit, and they I will merge.

Thanks for adding this! The sudden activity on this repo suggests its being used in some sort of project? Would be interesting to know more about that!

**Comment by garbit** [comment]
Hey, I've made the edits as a single commit.

We're looking at using the Onzo energy monitor in a research project at Newcastle University named [SunBlock](https://openlab.ncl.ac.uk/funding/sunblock-distributed-ledgers-and-decentralised-energy-in-sub-saharan-africa/) that's looking at using blockchain in off-grid solar energy networks in Tanzania. I'm a Research Associate working on the project and looking to create a collective virtual power plant to promote energy reduction.

Happy to chat more about the project just ping me an email - andy.garbett@newcastle.ac.uk

You're also welcome to get involved if you'd like an interesting project to work on :)

**Pull request merged**